### PR TITLE
Sprint 15 (dev->stg)

### PR DESCRIPTION
**Pull request by afrasier**
New features:
- When integrations load or update positions, SavedSearches will send notifications if new positions match their criteria ( #353 )

Tweaks:
- SOAP tweaks ( #346 )
- ADFS tweaks ( #349 )
- IE 11 middleware ( #351 )

Others:
- Deployment instructions ( #347 )

### CODE REVIEW

File: talentmap_api/common/middleware.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+class IE11Middleware:
+
```
Comment (jseppi) [inline talentmap_api/common/middleware.py]: A doc string about why this exists would be great

File: talentmap_api/integrations/synchronization_helpers.py [hunk 2]
```diff
@@ -386,16 +388,21 @@ def mode_positions():
         "effective_date": parse_date("effective_date"),
     }
 
-    return (soap_arguments, instance_tag, tag_map, collision_field, None, None)
+    def post_load_function(model, new_ids, updated_ids):
+        # If we have any new or updated positions, updated saved search counts
```
Comment (jseppi) [inline talentmap_api/integrations/synchronization_helpers.py]: `updated` -> `update`

File: talentmap_api/integrations/synchronization_helpers.py [hunk 3]
```diff
@@ -434,11 +441,22 @@ def mode_cycles():
 
     def override_loading_method(loader, tag, new_instances, updated_instances):
         # If our cycle exists, clear it's position numbers
```
Comment (jseppi) [inline talentmap_api/integrations/synchronization_helpers.py]: `it's` -> `its`

File: talentmap_api/integrations/synchronization_helpers.py [hunk 4]
```diff
@@ -462,6 +480,11 @@ def mode_cycle_positions():
 
     }
 
+    def post_load_function(model, new_ids, updated_ids):
+        # If we have any new or updated positions, updated saved search counts
```
Comment (jseppi) [inline talentmap_api/integrations/synchronization_helpers.py]: `updated` -> `update`

File: talentmap_api/saml2/attribute_maps/basic.py.bak [hunk 5]
```diff
@@ -0,0 +1,325 @@
+MAP = {
```
Comment (jseppi) [inline talentmap_api/saml2/attribute_maps/basic.py.bak]: Is this file needed?

Comment (afrasier) [reply talentmap_api/saml2/attribute_maps/basic.py.bak]: I wanted to hold on to it for reference, just in case.

Comment (burgwyn) [reply talentmap_api/saml2/attribute_maps/basic.py.bak]: Let’s keep it until we’re done with this work
